app/assets/javascripts/application.js

```diff
@@ -17,5 +17,15 @@
 //= require bootstrapValidator.min
 //= require pnotify
 //= require unobtrusive_flash
-//= require select2
+//= require select2-full
 //= require_tree .
+$(document).ready(function () {
```

Comment:
With the turbo-links enabled its good to use `$(document).on('turbolinks:load', function() {....`, this will ensure the scripts loads every time a page loads. Using `$(document).ready(function () {...` will not work after a page redirect, this is because with turbo-links, the page is loaded via ajax

Reply:
@mwenda-eric thanks for this insight `$(document).on('turbolinks:load', function()`
That issue has been giving me problems with javascript for a while